doc/environment_variables.md

```diff
@@ -0,0 +1,64 @@
+gRPC environment variables
+--------------------------
+
+gRPC C core based implementations (those contained in this repository) expose
+some configuration as environment variables that can be set.
+
+* GRPC_ABORT_ON_LEAKS
+  A debugging aid to cause a call to abort() when gRPC objects are leaked past
+  grpc_shutdown(). Set to 1 to cause the abort, if unset or 0 it does not
+  abort the process.
+
+* GRPC_GOOGLE_CREDENTIALS_ENV_VAR
```

Comment:
Did we change the name?
It used to be GOOGLE_APPLICATION_CREDENTIALS or sth in that sense.